services/client/src/components/header/styles/index.js

```diff
@@ -32,11 +32,14 @@ export default css`
     border-top-right-radius: 0.25rem;
     display: flex;
     padding: 0.5rem;
+    position: fixed;
```

Comment:
Applying `fixed` here causes the sticky headers to look a little bit off; It might be nicer if the sticky header was below the header, instead of overlaid: 
<img width="612" alt="screen shot 2018-09-14 at 20 29 54" src="https://user-images.githubusercontent.com/1443700/45571157-42fac480-b85d-11e8-80c2-9b23a0c84371.png">